packages/hash/frontend/src/graphql/queries/knowledge/entity.queries.ts

```diff
@@ -103,8 +102,7 @@ export const updatePersistedEntityMutation = gql`
       entityId: $entityId
       updatedProperties: $updatedProperties
     ) {
-      ...PersistedEntityFields
+      # This is a scalar, which has no selection.
```

Comment:
This mutation doesn't exist anymore, did you intend on replacing it with `updateEntityWithMetadata`?

Reply:
Yes, thanks this is intended to be replaced. I'm very confused as to why this isn't being picked up by codegen?
Anyway, I will update these but I expect the follow-up tasks to fix the frontend would make sure these queries are updated correctly: 3ec652e69ae70c16980a7b8a4e1b7ae13f803299

Reply:
> I'm very confused as to why this isn't being picked up by codegen?

To be honest, me too - is this potentially caching related as well?

Reply:
I don't think it's caching, running `yarn  graphql-codegen --config codegen.yml` from the frontend folder seems to provide the same output. Perhaps the query isn't used anywhere?